Update site with feedback on value prop and communication (#270)
2i2c-org/2i2c-brochure: content/_index.md

                  
      # Add your Hero text here
      text: |-

>         Our interactive computing platform gives research and education communities a digital home to create and share knowledge, and a global network of communities to learn from.

Reviewer: ```suggestion
        Our interactive computing platform gives research and education communities a digital home to create and share knowledge with a global network of communities to learn from.
```
Reduces the number of "and"s in one sentence.

  - block: markdown
    id: service
    content:
      title: We help communities build a home in the cloud for interactive computing
      subtitle: 
      text: |

        Our platform **empowers community leaders in research and education to design a hub** for their community to create and share knowledge. We do so by bringing together open tools, services, data, and infrastructure.

>         {{< servicetech >}}

Reviewer: This renders differently on Safari and Firefox (not checked Chrome).

<img width="1392" alt="Screenshot 2024-07-06 at 10 27 16" src="https://github.com/2i2c-org/2i2c-org.github.io/assets/45105935/803442fb-6066-467b-b291-c5a8bf436ef6">
<img width="1391" alt="Screenshot 2024-07-06 at 10 27 44" src="https://github.com/2i2c-org/2i2c-org.github.io/assets/45105935/241c5150-636c-4800-94ed-aa8da929b5a6">

      subtitle: 
      text: |

>         Our platform **empowers leaders in research and education to design a hub that helps their community create and share knowledge**. It does so by bringing together open tools and services, data, and computational resources.

Reviewer: ```suggestion
        Our platform **empowers leaders in research and education to design a hub that helps their community create and share knowledge**. We bring together open tools and services, data, and computational resources to co-create customized solutions.
```
I tried to bring a more active voice to this sentence.

      subtitle: ""
      text: ""
      items:

>         # LEAVE OUT PARTNERSHIP because this forces us to use 3 columns

Reviewer: 😅
